worker-document-interface/src/main/java/com/hpe/caf/worker/document/model/Task.java

```diff
@@ -64,4 +65,17 @@
      * @see Application#getService(Class) Application.getService()
      */
     <S> S getService(Class<S> service);
+
+    /**
+     * Allows to set additional response options for this task.
+     * @param queueName Queue name to post response message to.
+     * @param customData Custom data to set on a response message.
+     */
+    void setResponseOptions(String queueName, Map<String, String> customData);
```

Comment:
This method made sense by itself, but there doesn't seem to be much point in it given that you've had to introduce the Response object.  Why not just add a `setQueueName()` etc. to it?